test/controlplane/pod/hostport/generate.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+#
+# Generate the golden test files for the HostPort test.
+#
+
+set -eux
+
+dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+. "${dir}/../../k8s_versions.sh"
+
+export KUBECONFIG="${dir}/kubeconfig"
+
+: Start a kind cluster
+kind create cluster --config "${dir}/manifests/kind-config-1.26.yaml" --name hostport
+
+: Wait for service account to be created
+until kubectl get serviceaccount/default; do
+    sleep 5
+done
+
+: Preloading images
+kind load --name hostport docker-image "${cilium_container_repo}/${cilium_container_image}:${cilium_version}" || true
+kind load --name hostport docker-image "${cilium_container_repo}/${cilium_operator_container_image}:${cilium_version}" || true || true
+
+: Install cilium
+cilium install --wait
+
+: Dump the initial state
+kubectl get nodes,pods -o yaml > "${dir}/init.yaml"
```

Comment:
```suggestion
kubectl get nodes,ciliumnodes,pods -o yaml > "${dir}/init.yaml"
```

with your PR it fails in one endpoint to me, I regenerated and also added this , but I'm not sure this makes a difference

Reply:
I don't think this is necessary (getting ciliumnodes). Passes for me as is.